### ISSUE TITLE
Field's grid position per section

### ISSUE THREAD

**Issue by onca-vega** [tecnocen-com/yii2-formgenerator#7]
Actually I need a way to know the position of all fields in every section, so the user could give some basic styles to it's own form. 

Right now we may know the order, but this value is not enough to style the columns in every stage or if after an specific field we should give an end line and continue displaying with next fields on next line.

I'm thinking in 2 posible ways to solve this:

1) Saving an x and y position in every form_section_field row.

2) Saving one new value on a new column in every form_section, called something like "columns", where I can save the actual columns that would be showed on an specific section, and just if a field needs an endline, in its form_section_field columns, create a new one called something like "endline", that should be a boolean, that if is true will tell me that after this field I should insert an "enter" to continue displaying all the left fields.

I'll be pending on your answer. Best regards.

**Comment by Faryshta** [comment]
how would you go for intersectional columns? example:

    [name]        [lastname]
    [email]       [password]
    [      comment         ]

**Comment by onca-vega** [comment]
Thinking about your question, I got an idea for this, if we have the next example:

-> FORM ONE

- Section A:
<table>
  <tr>
    <td>Col 1</td>
    <td>Col 2</td>
    <td>Col 3</td>
  </tr>
  <tr>
    <td colspan="2">[name]</td>
    <td>[lastname]</td>
  </tr>
  <tr>
    <td></td>
    <td>[password]</td>
    <td></td>
  </tr>
  <tr>
    <td>[username]</td>
    <td>[email]</td>
    <td>[age]</td>
  </tr>
  <tr>
    <td></td>
    <td colspan="2">[abstract]</td>
  </tr>
  <tr>
    <td colspan="3">[personal comment]</td>
  </tr>
</table>

So for FORM ONE Section A we have 3 columns, and the position definition of every field is:
- name ->  x:0, y:0, length: 2
- lastname ->  x:2, y:0, length: 1
- password ->  x:1, y:1, length: 1
- username -> x:0, y:2, length: 1
- email -> x:1, y:2, length: 1
- age -> x:2, y:2, length: 1
- abstract -> x:1, y:3, length: 2
- personal comment ->  x:0, y:4, length: 3

So, after this example we can see that the main information to support this styling issues is:
1) The columns of every form_section
2) X and Y positions, and length of every form_section_field

In other words, the solution for this is a sort of mixure of my two previous reviews.
Again, I'll be pending on your answer.

**Comment by neverabe** [comment]
@Faryshta, any news about this issue.....

**Comment by Faryshta** [comment]
i think this is outside the scope of formgenerator and should be handled as a new library, like formmakeup or something like that specially because this approach is not mobile friendly.
